shopfloor_mobile_base/static/wms/src/services/translation_registry.js

```diff
@@ -3,6 +3,17 @@
  * @author Simone Orsi <simahawk@gmail.com>
  * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
  */
+export function loadTranslation(callback, url) {
```

Comment:
Not really limited to translations, it could be moved elsewhere too. I'd leave it here but change the name:
```suggestion
export function loadJSON(callback, url) {
```